mealie/services/recipe/recipe_data_service.py

```diff
@@ -108,6 +108,14 @@ def write_image(self, file_data: bytes | Path, extension: str, image_dir: Path |
 
         return image_path
 
+    def delete_image(self) -> None:
+        image_dir = self.dir_image
+
+        try:
+            shutil.rmtree(image_dir)
+        except Exception as e:
+            self.logger.exception(f"Failed to delete image data: {e}")
```

Comment:
You can leave off the exception, `logger.exception` already logs the full traceback

```suggestion
            self.logger.exception(f"Failed to delete image data")
```

Reply:
Thanks!👍